cmd/kube-scheduler/app/server.go

```diff
@@ -190,10 +190,11 @@ func Run(cc schedulerserverconfig.CompletedConfig, stopCh <-chan struct{}) error
 
 	// Prepare the event broadcaster.
 	if cc.Broadcaster != nil && cc.EventClient != nil {
-		cc.Broadcaster.StartLogging(klog.V(6).Infof)
-		cc.Broadcaster.StartRecordingToSink(&v1core.EventSinkImpl{Interface: cc.EventClient.Events("")})
+		cc.Broadcaster.StartRecordingToSink(stopCh)
+	}
+	if cc.LeaderElectionBroadcaster != nil && cc.CoreEventClient != nil {
```

Comment:
I don't see cc.LeaderElectionBroadcaster to be set anywhere in the code - seems like a bug to me.

Reply:
oh good catch !